views/partner/partner.hbs

```diff
@@ -59,9 +59,9 @@
 					</div>
 				</div>
 				<div class="buttonWrapper">
-					<div class="btn toggle-partner" class="btn btn-info collapsed" data-toggle="collapse" data-target="#hiddenAdvisers">
+					<a href="#0" tabindex="0" class="btn toggle-partner" class="btn btn-info collapsed" data-toggle="collapse" data-target="#hiddenAdvisers">
```

Comment:
if you use a then I think you don't need tabindex because anchor tags are tabbable out of the box.

Reply:
I used a button instead